SharpMUSH.Client/Pages/DynamicApplication.razor.css

```
@@ -0,0 +1,97 @@
+    color: var(--accent);
+    margin-bottom: 0.5rem;
+}
+
+.app-title {
+    margin: 0;
+    font-family: var(--font-display);
+    font-size: 1.625rem;
+    font-weight: 600;
+    color: var(--text);
+    line-height: 1.15;
+    word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)